docs/analyzer/user_guide.md

```diff
@@ -1243,7 +1244,7 @@ cross translation unit analysis arguments:
                         analysis, using already available extra files in
                         '<OUTPUT_DIR>/ctu-dir'. (These files will not be
                         cleaned up in this mode.)
-  --ctu-on-the-fly      If specified, the 'collect' phase will not create the
+  --ctu-on-demand       If specified, the 'collect' phase will not create the
```

Comment:
Update the docs with the `--ctu-ast-loading-mode` help.

Reply:
Thanks for catching these! fixed the docs 📖 .